tensorflow/inference/docker/2.10/py3/sdk2.8.0/Dockerfile.neuron

```diff
@@ -76,7 +76,7 @@ RUN wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEU
 RUN apt-get update
 
 RUN apt-get install -y \
-    tensorflow-model-server-neuron=2.8.* \
+    tensorflow-model-server-neuron=1.15.0.2.3.0.0 \
```

Comment:
Does this need to be 2.10?

Reply:
Updated with the latest tensorflow 2.10-compatible release (which is now neuronx but still compatible)